### DISCUSSION TITLE
Unknown Problem

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#7184] [General] [answered]
Has anybody else been seeing alot more "Unknown Problem Somewhere In The System" since the last update?

**Comment by dougburks** [comment, marked as answer]
Have you drilled into the alerts to see the exact log that caused the alert?

Is it possible that the logs you're seeing are just for the update process itself?

**Reply by Acewiza** [reply]
It must be just something changing on the last update.  This is a small network, with nothing ever really going on except for occasional updates.  The unexpected unknowns seem fairly constant/consistent with nothing else unusual noted.  They all look like this:

`Feb 13 13:20:36 onion2 launcher[717]: {"caller":"distributed.cpp:144","component":"osquery","level":"stderr","msg":"E0213 13:20:36.533423  1214 distributed.cpp:144] Error executing distributed query: fleet_detail_query_munki_info: no such table: munki_info","severity":"info","ts":"2022-02-13T13:20:36.533447029Z"}'

**Reply by Acewiza** [reply]
Another interesting oddity possibly related to that .100 update is disk space utilization continues rising (good), but total logs reflected on the Kibana overview is steadily dropping.  I feel like this is related.

**Reply by Acewiza** [reply]
And the error logs are not (sorry) all looking like above, just the ones I saw yesterday:

'Feb 14 14:34:58 onion2 launcher[717]: {"caller":"query_target.go:75","err":"fetching target memberships: rpc error: code = Internal desc = stream terminated by RST_STREAM with error code: INTERNAL_ERROR","msg":"updating kolide_target_membership data","severity":"error","ts":"2022-02-14T14:34:58.958147065Z"}'

I'll see if I can find some more...

**Reply by dougburks** [reply]
```
Feb 13 13:20:36 onion2 launcher[717]: {"caller":"distributed.cpp:144","component":"osquery","level":"stderr","msg":"E0213 13:20:36.533423 1214 distributed.cpp:144] Error executing distributed query: fleet_detail_query_munki_info: no such table: munki_info","severity":"info","ts":"2022-02-13T13:20:36.533447029Z"}
```
```
Feb 14 14:34:58 onion2 launcher[717]: {"caller":"query_target.go:75","err":"fetching target memberships: rpc error: code = Internal desc = stream terminated by RST_STREAM with error code: INTERNAL_ERROR","msg":"updating kolide_target_membership data","severity":"error","ts":"2022-02-14T14:34:58.958147065Z"}
```
Both of these logs appear to be related to Fleet/osquery. Are these services working correctly for you?

**Reply by Acewiza** [reply]
'Feb 14 17:49:32 onion2 launcher[717]: {"caller":"distributed.cpp:144","component":"osquery","level":"stderr","msg":"E0214 17:49:32.870034  1214 distributed.cpp:144] Error executing distributed query: fleet_detail_query_munki_info: no such table: munki_info","severity":"info","ts":"2022-02-14T17:49:32.870123324Z"}'

Slightly different.  Thinks it's not finding a table.

**Reply by Acewiza** [reply]
Log count rising again, maybe burped something the past few days.  Let's forget about that little misdirection, for now.

**Reply by Acewiza** [reply]
Lotsa table finding issues.

`Feb 15 16:13:22 onion2 launcher[794]: {"caller":"distributed.cpp:144","component":"osquery","level":"stderr","msg":"E0215 16:13:22.070667  1271 distributed.cpp:144] Error executing distributed query: fleet_detail_query_google_chrome_profiles: no such table: google_chrome_profiles","severity":"info","ts":"2022-02-15T16:13:22.070765441Z"}`

**Reply by defensivedepth** [reply]
These types of logs can be ignored at this point.

**Reply by dougburks** [reply]
If necessary, you could tune your Wazuh ruleset to avoid firing an alert on these nuisance logs:
https://docs.securityonion.net/en/2.3/wazuh.html#tuning-rules

**Reply by Acewiza** [reply]
Nuisance logs!  I knew there had to be some explanation.  I'm sure you guys have more important logs to worry about -thanks!  ;-)

**Comment by Acewiza** [comment]
I believe so, I mean so-status reports all OK, and like I said, no other
real indication anything is broken.

On Tue, Feb 15, 2022 at 4:52 AM Doug Burks ***@***.***> wrote:

> Feb 13 13:20:36 onion2 launcher[717]: {"caller":"distributed.cpp:144","component":"osquery","level":"stderr","msg":"E0213 13:20:36.533423 1214 distributed.cpp:144] Error executing distributed query: fleet_detail_query_munki_info: no such table: munki_info","severity":"info","ts":"2022-02-13T13:20:36.533447029Z"}
>
> Feb 14 14:34:58 onion2 launcher[717]: {"caller":"query_target.go:75","err":"fetching target memberships: rpc error: code = Internal desc = stream terminated by RST_STREAM with error code: INTERNAL_ERROR","msg":"updating kolide_target_membership data","severity":"error","ts":"2022-02-14T14:34:58.958147065Z"}
>
> Both of these logs appear to be related to Fleet/osquery. Are these
> services working correctly for you?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/7184#discussioncomment-2180370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASJEPQQHVZCN4MPZBKGJXDLU3I463ANCNFSM5OETNFQA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> <Security-Onion-Solutions/securityonion/repo-discussions/7184/comments/2180370
> @github.com>
>


-- 
Paul D. Shaffer, MSgt, USAF (ret.)
